README.md

```diff
@@ -18,10 +18,12 @@
 </p>
 <br>
 
-Go DTLS 1.2 implementation. The original user is pion-WebRTC, but we would love to see it work for everyone.
+Native [DTLS 1.2](rfc6347) implementation in the Go programming language.
```

Comment:
Link is not correctly recognized.
See: https://github.com/pion/dtls/blob/ed25519/README.md

Maybe
```suggestion
Native [DTLS 1.2][rfc6347] implementation in the Go programming language.
```

Reply:
Ah crap, thanks for catching that. I missed it b/c GitHub still renders it, but does the wrong thing :man_facepalming: